src/app/components/BlockChainLinks.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Snippet } from "@nextui-org/react";
+
+const BlockchainLinks = ({ link }: { link: string }) => {
+  if (!link) return null;
```

Comment:
This check should come outside of BlockchainLinks, if there is no link present then you should not render BlockchainLinks. Instead of loading it and then returning null from it.